docs/FAQ/index.md

```diff
@@ -28,135 +36,138 @@ directory with product configuration and user groups in the Admin Console
+no check on the domain?
+
+User Sync uses the UMAPI, so it can both query and manage the user 
 groups and product access for users in both owned and accessed domains.  However, 
 like the Admin Console, User Sync can only be used to create and update user 
 accounts in owned domains, not domains owned by other organizations.
 
-**Q: Is there an update function, or just add/remove users (for only 
-federatedID)?**<br/>
-A: For all types of ID (Adobe, Enterprise, and Federated), User Sync supports 
+### Is there an update function, or just add/remove users (for only 
+federatedID)?
+
+For all types of ID (Adobe, Enterprise, and Federated), User Sync supports 
 update of group memberships under control of the --process-groups option.  
 For Enteprise and Federated IDs, User Sync supports update of first name, last 
 name, and email fields under control of the --update-user-info  option.  When 
 country updates become available in the Admin Console, they will also be 
 available via the UMAPI.  And for Federated IDs whose "User Login Setting" 
 is "Username", User Sync supports update of username as well as the other fields. 
 
-**Q: The user sync tool is dedicated to a particular OS?** (for example, win package 
-is different than unix or linux installer?)<br/>
-A:  User Sync is an open source python project.  Users can build for any OS platform they desire.  We provide builds for Windows, OS X, Ubuntu, and Cent OS 7 platforms.
+### The user sync tool is dedicated to a particular OS? (for example, win package 
+is different than unix or linux installer?)
+
+User Sync is an open source python project.  Users can build for any OS platform they desire.  We provide builds for Windows, OS X, Ubuntu, and Cent OS 7 platforms.
  
-**Q: Has this been tested on python 3.5?**<br/>
-A: User Sync has been run successfully on Python 3.x, but most of our use and testing is on Python 2.7 so you may discover problems, and we only provide builds on Python 2.7.  Feel free to report (and contribute fixes) to the open source site at https://github.com/adobe-apiplatform/user-sync.py.
+### Has this been tested on python 3.5?
 
-**Q: If something changes in the API (new field in creating users, for example ) how will the update  be applied to the user sync tool?** New package or python modules can be directly modified customer side?<br/>
-A: User sync is an open source project.  Users can download and build the latest 
+User Sync has been run successfully on Python 3.x, but most of our use and testing is on Python 2.7 so you may discover problems, and we only provide builds on Python 2.7.  Feel free to report (and contribute fixes) to the open source site at https://github.com/adobe-apiplatform/user-sync.py.
+
+### If something changes in the API (new field in creating users, for example ) how will the update  be applied to the user sync tool?
+
+User sync is an open source project.  Users can download and build the latest 
```

Comment:
I think you should say that User Sync is built on top of umapi-client, which is the only module with direct knowledge of the API.  When the API changes, umapi-client always gets updated to support it.  If and when API changes provide for more User Sync-related capabilities, then User Sync may be updated to provide them.